test/unit/connect-inject-clusterrolebinding.bats

```diff
@@ -16,6 +16,7 @@ load _helpers
   local actual=$(helm template \
       -x templates/connect-inject-clusterrolebinding.yaml  \
       --set 'global.enabled=false' \
+      --set 'client.enabled=true' \
```

Comment:
These had to be set to true because otherwise the connect-inject-* templates don't render now.